package.json

```diff
@@ -45,17 +50,23 @@
     "brace": "^0.8.0",
     "browserify": "^13.0.0",
     "csslint": "^1.0.2",
+    "env-cmd": "github:serapath-contribution/env-cmd",
```

Comment:
yes - i plan to change it. I made a pull request to the original `env-cmd` module and pinged the author. since then, the author already contacted me that he likes the change and i am awaiting a merge. Once it's merged in, i'd change that dependency to the npm module `[env-cmd](https://www.npmjs.com/package/env-cmd)` ... so yes - it's meant to be temporary :P